src/app/views/products/product.controller.coffee

```diff
@@ -66,7 +66,7 @@
 
   vm.updateProductPricing = (pricingPlan) ->
     # Add any unsaved prices
-    unless vm.priceForm.$invalid
+    if vm.priceForm && !vm.priceForm.$invalid
```

Comment:
Let's make this `if pricingPlan && !vm.priceForm.$invalid` just so the reason for the check is more obvious since it's being passed to `vm.addPrice` below.